cmd/bacalhau/docker_run.go

```diff
@@ -262,44 +270,57 @@ var dockerRunCmd = &cobra.Command{
 				return err
 			}
 		}
+		if isLocal {
+			client, err := local.NewDockerClient()
```

Comment:
I think we need to check `waitForJobToFinish` and `waitForJobToFinishAndPrintOutput` here, like the branch below.
This should bring in support for `--wait` and `--download`  flags

Reply:
I think we don't need that as it directly prints output to the console, here its more like docker run command which print stdout,err to console

Reply:
Oh. Do you mean checking that is not needed because it already prints out to console?